Handle multiple creators in Paella player

lkiesow opened the pull request:
This patch fixes the problem that the Paella player would just display a
single random creator, even if the event has multiple creators.

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] be against the correct branch (features can only go into develop)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

lkiesow committed:
Handle multiple creators in Paella player

This patch fixes the problem that the Paella player would just display a
single random creator, even if the event has multiple creators.

Arnei commented on modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.descriptionPlugin/mh_description.js:
```diff
@@ -61,7 +61,12 @@ paella.addPlugin(function() {
+          : [thisClass._episode.mediapackage.creators.creator])
+          .join(', ');
+      }
       if (thisClass._episode.dcContributor) { this.desc.contributor = thisClass._episode.dcContributor; }
```
Not necessary, but it might be nice to also help out multiple contributors while you're at it.
```suggestion
      if (thisClass._episode.dcContributor) { this.desc.contributor = thisClass._episode.dcContributor; }
            if (thisClass._episode.dcContributor) {
        this.desc.contributor = (Array.isArray(thisClass._episode.mediapackage.contributors.contributor)
          ? thisClass._episode.mediapackage.contributors.contributor
          : [thisClass._episode.mediapackage.contributors.contributor])
          .join(', ');
      }
```

lkiesow replied:
I think you meant this, right?

```suggestion
      if (thisClass._episode.mediapackage.contributors) {
        this.desc.contributor = (Array.isArray(thisClass._episode.mediapackage.contributors.contributor)
          ? thisClass._episode.mediapackage.contributors.contributor
          : [thisClass._episode.mediapackage.contributors.contributor])
          .join(', ');
      }
```

Arnei replied:
Of course, I meant what you wrote. My apologies.

Arnei reviewed (approved):
Works in my local AllInOne Opencast 9 installation.
The suggestion below does not need to be implemented for my approval, it's more of a nice-to-have.

lkiesow committed:
Handle multiple creators in Paella player

This patch fixes the problem that the Paella player would just display a single random contributor, even if the event has multiple contributors.
